src/NServiceBus.Hosting.Windows/EndpointNameAttribute.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace NServiceBus.Hosting.Windows
+{
+    using System;
+
+    /// <summary>
+    /// Used to specify the name of the current endpoint.
+    /// Will be used as the name of the input queue as well.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class)]
+    public sealed class EndpointNameAttribute : Attribute
```

Comment:
I want planning on removing this from the host atm. The core one will get an obsolete.

This one is tied in at a very low level to detecting endpoint name at install time. I did not intend to touch that code